### ISSUE TITLE
the container paused has no status while I using "kata-runtime list "

### ISSUE THREAD

**Issue by hanwei1990** [kata-containers/runtime#399]
# Description of problem

root@host-192-168-200-145:~# kata-runtime -version
kata-runtime  : 1.0.0
   commit   : # <<unknown>>
   OCI specs: 1.0.1

root@host-192-168-200-145:~# kata-runtime pause e6b4058051037c5a2dba7d5039e1e6117e72edac9b7e6a169636c8258f8f35f8

# Expected result

root@host-192-168-200-145:~# kata-runtime list
ID                                                                 PID         STATUS      BUNDLE                                                                                                                               CREATED                          OWNER
e6b4058051037c5a2dba7d5039e1e6117e72edac9b7e6a169636c8258f8f35f8   3299            **paused**            /run/docker/containerd/daemon/io.containerd.runtime.v1.linux/moby/e6b4058051037c5a2dba7d5039e1e6117e72edac9b7e6a169636c8258f8f35f8   2018-06-13T02:25:14.681031453Z   #0

# Actual result

root@host-192-168-200-145:~# kata-runtime list
ID                                                                 PID         STATUS      BUNDLE                                                                                                                               CREATED                          OWNER
e6b4058051037c5a2dba7d5039e1e6117e72edac9b7e6a169636c8258f8f35f8   3299                    /run/docker/containerd/daemon/io.containerd.runtime.v1.linux/moby/e6b4058051037c5a2dba7d5039e1e6117e72edac9b7e6a169636c8258f8f35f8   2018-06-13T02:25:14.681031453Z   #0

**Comment by jodh-intel** [comment]
Hi @hanwei1990 - thanks for reporting.

Please can you run `sudo kata-collect-data.sh`, check the output to ensure there is nothing confidential/secret in it, then paste into a comment here? That would give us a better indication of the issue you are seeing.

**Comment by hanwei1990** [comment]
@jodh-intel I'm sorry that I can't find the kata-collect-data.sh. Cloud you show me where I can find the script?

**Comment by jodh-intel** [comment]
Oh - sorry @hanwei1990. I realise now that you've hit a packaging issue where that script wasn't included (https://github.com/kata-containers/packaging/issues/48).

Here's how to get the script:

```
$ go get -d github.com/kata-containers/runtime
$ cd $GOPATH/src/github.com/kata-containers/runtime
$ make
```

Then, to run it:

```
$ sudo bash data/kata-collect-data.sh > /tmp/kata.log
```

**Comment by hanwei1990** [comment]
[kata.log](https://github.com/kata-containers/runtime/files/2102084/kata.log)
@jodh-intel please take a look at the log

**Comment by jodh-intel** [comment]
Hi @hanwei1990 - I think the problem is that pause/resume were not implemented in the agent version bundled with 1.0.0. That change landed *just* after the release:

- https://github.com/kata-containers/agent/pull/246

It will of course be included in the next release (/cc @egernst, @jcvenegas).

___

If you want to use this feature in the meantime, you could use our [`osbuilder`](https://github.com/kata-containers/osbuilder) tool to create an image with the latest agent code:

- https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#create-and-install-rootfs-and-initrd-image

You'll need to follow these steps to get the latest agent into an image:

- https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#build-a-custom-kata-agent---optional
- https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#add-a-custom-agent-to-the-image---optional
